apps/web/pages/apps/installed/[category].tsx

```diff
@@ -198,31 +200,29 @@ const IntegrationsContainer = ({ variant, exclude }: IntegrationsContainerProps)
 };
 
 const querySchema = z.object({
-  category: z.nativeEnum(InstalledAppVariants),
+  category: z.enum(InstalledAppVariants),
```

Comment:
Does this work? I had problems in the past trying to do this, always had to use `nativeEnum`

Reply:
Saw the change in the enum itself, I get it 👍